components/root/root.jsx

```diff
@@ -389,6 +391,18 @@ export default class Root extends React.Component {
                         path={['/_redirect/integrations*', '/_redirect/pl/:postid']}
                         component={PermalinkRedirector}
                     />
+                    {this.props.plugins?.map((plugin) => (
+                        <Route
+                            key={plugin.id}
+                            path={'/plug/' + plugin.route}
```

Comment:
Do we still need `/plug`?

Reply:
Yes. We don't want plugin IDs to conflict with team names.